radsat-sk/operation/subsystems/RADCS.c

```diff
@@ -0,0 +1,303 @@
+/**
+ * @file RADCS.c
+ * @date March 26, 2022
+ * @author Addi Amaya (Caa746)
+ */
+
+#include <RUart.h>
+#include <RADCS.h>
+#include <RCommon.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+/***************************************************************************************************
+                                            DEFINITIONS
+***************************************************************************************************/
+
+#define TELECOMMAND_20    	          	((uint8_t) 0x14)
+
+#define TELECOMMAND_20_LEN              ((uint8_t) 3)
+#define TELECOMMAND_RESPONSE_LEN		((uint16_t)1)
+
+#define TELECOMMAND_OFFSET_ID           ((uint8_t) 2)
+#define TELECOMMAND_OFFSET_0            ((uint8_t) 3)
+#define TELECOMMAND_OFFSET_1            ((uint8_t) 4)
+#define TELECOMMAND_REPONSE_OFFSET		((uint8_t) 2)
+
+#define TELEMETRY_22                    ((uint8_t) 0x96)
+#define TELEMETRY_25                    ((uint8_t) 0x99)
+
+#define TELEMETRY_22_LEN				((uint8_t) 10)
+#define TELEMETRY_25_LEN				((uint8_t) 10)
+#define TELEMETRY_REQUEST_LEN			((uint8_t) 1)
+#define TELEMETRY_REPLY_SIZE_6			((uint8_t) 6)
+
+#define TELEMETRY_OFFSET_0              ((uint8_t) 2)
+#define TELEMETRY_OFFSET_2              ((uint8_t) 4)
+#define TELEMETRY_OFFSET_4              ((uint8_t) 6)
+#define TELEMETRY_OFFSET_5              ((uint8_t) 7)
+#define TELEMETRY_ID_OFFSET		        ((uint8_t) 2)
+
+#define BASE_MESSAGE_LEN				((uint8_t) 4)
+#define START_IDENTIFIER1               ((uint16_t) 0x1F)
+#define START_IDENTIFIER2               ((uint16_t) 0x7F)
+#define FILLER							((uint16_t) 0x00)
+#define END_IDENTIFIER1                 ((uint16_t) 0x1F)
+#define END_IDENTIFIER2                 ((uint16_t) 0xFF)
+
+#define SUN_SENSOR	         	        ((uint8_t) 0)
+#define IMAGE_SENSOR	                ((uint8_t) 1)
+#define SRAM1                           ((uint8_t) 0)
+#define SRAM2                           ((uint8_t) 1)
```

Comment:
[No space after a cast](https://github.com/USST-RADSAT-SK/software-and-command#naming-1)